sdk/typescript/examples/01-block-unsafe-tool/README.md

```
@@ -0,0 +1,32 @@
+Show JamJet's policy enforcement blocking a destructive tool call before execution.
+
+## Run
+
+```bash
+pnpm install
+pnpm start
+```
+
+## Expected output
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)